### DISCUSSION TITLE
Big Thanks

### DISCUSSION THREAD

**Discussion by Cobolt78** [mikiher/audiobookshelf-windows#4] [General]
I just wanted to say Thank you so much to the team for the windows version of audiobookshelf.
I have been running an audiobookshelf server on my synology nas for about a year and a half now and it has been great. The only downsides is it was very slow to load server data onto clients (sometimes over 1 minute, also searching was very slow as well). 
I decided to make another test server running docker for windows and install audiobookshelf server and had mixed results, I actually tested it on a second windows machine and experienced the same issue as on the first machine, which was for some reason when you first try and connect to the server it would timeout but if you then closed the app down on android or re-open the page again in a browser it would connect and load the server data straight away, Once you were connected, loading, searching, browsing was very quick compared to my NAS. This was not ideal to tell family and friends to open the app, wait for it to time out then close the client and retry.

I didn't even realise there was a windows install in beta to test so I was stoked to give it a try, I knew this was a beta so my expectations wasn't too high but so far I am blown away, everything works great. The only small issue I have found so far is long file names and the error location doesnt exist because the filenames are too long, I enabled long filenames in windows via the registry and restarted but still got the same error. So I just renamed the file and folders so it was shorter and all problems solved.

Just thought I would share my experience so far with the windows install and to thank you immensly to the team for all your hard work, If only I could buy you all a beer, wine, spirit or a coffee hahahaha.

Thanks again
:)

**Comment by mikiher** [comment]
Thanks for trying the beta, and I'm happy it's working well for you.

Can I ask you to please open a bug regarding the long filename issue, and describe the settings exactly (is it the data directory path that is too long? where exactly did you get the "location doesnt exist" error?)
